image-loader/app/controllers/ImageLoaderController.scala

```diff
@@ -143,48 +156,31 @@ class ImageLoaderController(auth: Authentication, downloader: Downloader, store:
     }
   }
 
-  private def projectS3ImageById(imageId: String): Try[Option[Image]] = {
-    Logger.info(s"projecting image: $imageId")
+  private def projectS3ImageById(imageId: String, requestLoggingContext: RequestLoggingContext): Try[Option[Image]] = {
+    Logger.info(s"projecting image: $imageId")(requestLoggingContext.toMarker())
 
     import ImageIngestOperations.fileKeyFromId
     val s3Key = fileKeyFromId(imageId)
     val s3 = S3Ops.buildS3Client(config)
 
     if (!s3.doesObjectExist(config.imageBucket, s3Key)) return Try(None)
 
-    Logger.info(s"object exists, getting s3 object at s3://${config.imageBucket}/$s3Key to perform Image projection")
+    Logger.info(s"object exists, getting s3 object at s3://${config.imageBucket}/$s3Key to perform Image projection")(requestLoggingContext.toMarker())
 
     val s3Source = s3.getObject(config.imageBucket, s3Key)
-    val lastModified = s3Source.getObjectMetadata.getLastModified.toInstant.toString
```

Comment:
I think an [`apply` method](https://github.com/guardian/grid/pull/2813/files#diff-f0b19673766219fe5248861e4378febcR35) is more readable here.